nginx/opt/containerbuddy/nginx.json

```diff
@@ -16,5 +18,50 @@
       "poll": 7,
       "onChange": "/opt/containerbuddy/reload-nginx.sh"
     }
-  ]
+  ],
+  "telemetry": {
+    "name": "telemetry",
+    "url": "/telemetry",
+    "port": 9090,
+    "ttl": 30,
+    "poll": 10,
```

Comment:
The answer is in https://github.com/tgross/containerbuddy/blob/gh27_metrics/telemetry/README.md#configuring-sensors.

Similar question to the above: is it important to make this a user-configurable setting? Previous conversation: https://github.com/joyent/containerbuddy/issues/27#issuecomment-199255902